### ISSUE TITLE
Failure increasing bounty amount

### ISSUE THREAD

**Issue by danlipert** [gitcoinco/web#6746]
I was attempting to increase the amount on an already active bounty. https://gitcoin.co/issue/MyBitFoundation/MyBit-Go.app/545/4395
And the first transaction failed, but still took the fee for gitcoin.
Failed tx: https://etherscan.io/tx/0x6295ecde7b5902ba728629be4199598d71afbeb09bdd8bb7acd49a6e113217ca
Fee taken : https://etherscan.io/tx/0xf89f888652e194d13609dab86fab9758b33862e9c097e2b94a52cb84a8f3246b
And I tried again bc I didn't realise the fee had been taken and same thing happened.
Failed tx: https://etherscan.io/tx/0x881dae0e2d031dd3ca97eaae907923f3e64bf75d55db238890d5e6a7059b2715
Fee Taken : https://etherscan.io/tx/0x36f853cf3d1a261b708006ac7e1f509a304ec511c873b861855c3118d78f1b0f
And a third fee was taken:
https://etherscan.io/tx/0x93c7877af67d5078396cc12b1c798710f67f1b09c3986fa0e7bc0ad419959ac7
So to summarise, 3 times the increase bounty amount function failed, but the gitcoin fees were still withdrawn from my account.

**Comment by thelostone-mc** [comment]
locally just tested it and seemed to work fine 
![image](https://user-images.githubusercontent.com/5358146/83510608-e8027880-a4ea-11ea-86cf-a43dce63101b.png)

**Comment by thelostone-mc** [comment]
Txn: https://etherscan.io/tx/0x881dae0e2d031dd3ca97eaae907923f3e64bf75d55db238890d5e6a7059b2715

Bounty: https://gitcoin.co/issue/MyBitFoundation/MyBit-Go.app/545/4395
Bounty Amount: 4200 DAI
Increased funding by: 1070 DAI
